app/views/entities/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Entities</h1>
+
+<div id="entities">
+  <% @entities.each do |entity| %>
+    <%= render entity %>
```

Comment:
- [ ] Please ensure that the list of transactions is ordered by the most recent :pray:

Reply:
The code in the _entities.htm.erb I will fix it in a while